tests/e2e/support/constants.js

```diff
@@ -7,6 +7,11 @@ export const defaultViewportDimensions = {
   height: 768,
 };
 
+/**
```

Comment:
Using this file feels a bit strange to me - I don't want to have these strings proliferate all over the place, but keeping this in  the `tests` dir is also wrong. Where would we like to move this to?

Reply:
I like the idea of moving it. Maybe it can go into `src/components/nodes/nodeTypes.js`.

This might tie in nicely with the refactor I was thinking about for https://github.com/ProcessMaker/modeler/pull/1099#issuecomment-587941206. If we extract this to a "node map", it can be used in the tests, it can be used to create the various node classes I'm thinking of (if we specify a type and a class property, not just the string), and it can be used to replace the string literals that are spread all over the place.

There are some files where this ID is exported, e.g.:
https://github.com/ProcessMaker/modeler/blob/c5c61d2e427ccf9186b501bf3f3d027c42f045d9/src/components/nodes/pool/index.js#L5

But I found that keeping it there caused circular dependency issues; so moving these IDs to some central file to be used by all would be great.

Reply:
Wanna pair on this to make sure we align our two refactors? I think this would be the last change and then this PR can come out of draft.